src/models/ISigInParams.ts

```diff
@@ -0,0 +1,6 @@
+import { InputName } from '../utils/consts';
+
+export interface ISigInParams {
```

Comment:
❗️
И так везде, где есть эта очепятка
```suggestion
export interface ISignInParams {
```

Reply:
Согласен! Исправил !